# Codes

This file includes the code for each figure, table and supplementary material. All the code is performed by R language (Version 3.5.1). The expected output have been saved in an independent folder named as "Output" in the "Source_data" folder.
The expected run time for the code of each figure or table is less than 10 minutes.

Each R file (code file) includes the description of its function.

## Note:
1. Figure 1A was drawn manually based on the source data.\
2. Figure 3C was drawn by Cytoscape software (Version 3.6.1). In source file "Figure_3C_GoTerm_inform.txt", We took the column "Term_name" as node and "Label" as node attribute. The size of each node (go term) depends on the number of genes enriched in this go term (column "size").In the column "Label",the overlapped go terms identified from TCGA and Japanese cohort were denoted as 1. Among these overlapped go terms, the go terms which were associated with the four prognostic transcripts were denoted as 4. The go terms exclusively identified from TCGA cohort were denoted as 2.The go terms exclusively identified form Japanese cohort were denoted as 3.\
3. The Figure 4A was drawn manually based on the source data.\
4. For Figure 4B, the homology models were built using Schrodinger Suite software (SchrË†dinger/2019-3, LLC, New York, NY). \
5. The Figure 4C, 4D and 5A were extracted from raw Western blots.\
6. The Figure 5B-5D were drawn by excell based on the source data.\
7. The Figure S3-S5 were drawn by excell based on the source data.\
8. For Table S14, the amino acid sequences of different transcripts were aligned using Uniprot website.\
9. For Table S15,the quantification of protein is based on the western blots image.\


